torch_xla/csrc/tensor_methods.cpp

```diff
@@ -1399,6 +1400,16 @@ XLATensor XLATensor::log_softmax_backward(const XLATensor& grad_output,
       grad_output.GetIrValue(), output.GetIrValue(), dim));
 }
 
+XLATensor XLATensor::logsumexp(const XLATensor& input,
```

Comment:
This should be moved down. Functions (which are ops implementation) must be sorted alphabetically.

Reply:
Yup, I first name this log_sum_exp and forgot to reorder functions after rename it.